README.md

```diff
@@ -114,13 +115,39 @@ an API-key and the resource type. To read from a table you should also specify t
 
 ### Writing
 
-To write to a resource you'll need to register a DataFrame that was read from
+There are two ways you can use cdp-spark-datasource to write to CDF: using `insertInto` or using the `save` function. 
```

Comment:
Should probably still use "CDP" here and rename it to "CDF" everywhere in a separate commit.

Reply:
Done!